### PR TITLE
Reverse sign

### LINKED ISSUE #22: Add the change sign button functionality
It should reverse the sign of the currently displayed number.

### PR DESCRIPTION
**Pull request by EdiOancea**
Closes #22

### CODE REVIEW

**Comment by EdiOancea** [comment]
@lgnstn  updated

File: src/components/Calculator.js [hunk 1]
```diff
@@ -30,13 +47,36 @@ class Calculator extends Component {
...
+      if (a[0] !== '0') {
+        if( a[0] !== '-') {
+          this.setState({ a: '-'.concat(a), displayed: '-'.concat(a) })
+        } else {
+          this.setState({ a: a.substring(1, a.length), displayed: a.substring(1, a.length) })
+        }
+      }
+    } else {
+      if (b[0] !== '0') {
+        if( b[0] !== '-') {
```
Comment (lgnstn) [inline src/components/Calculator.js]: Spaces.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -30,13 +47,36 @@ class Calculator extends Component {
     }
   }
 
+  reverseSign() {
+    const { a, b, operation } = this.state;
+    if (operation === '') {
+      if (a[0] !== '0') {
+        if( a[0] !== '-') {
```
Comment (lgnstn) [inline src/components/Calculator.js]: Spaces.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -30,13 +47,36 @@ class Calculator extends Component {
...
+          this.setState({ a: '-'.concat(a), displayed: '-'.concat(a) })
+        } else {
+          this.setState({ a: a.substring(1, a.length), displayed: a.substring(1, a.length) })
```
Comment (lgnstn) [inline src/components/Calculator.js]: `;`.

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Pull request merged**